src/Fixers.php

```diff
@@ -122,93 +196,198 @@ class Fixers
      */
     public static $risky = array(
         'ereg_to_preg',
-        'php4_constructor',
+        'no_php4_constructor',
         'php_unit_construct',
+        'php_unit_dedicate_assert',
         'php_unit_strict',
         'psr0',
         'psr4',
-        'strict',
+        'strict_comparison',
         'strict_param',
     );
 
 
     /**
      * @var string[]
      */
+    public static $aliases = array(
+        'operators_spaces' => 'binary_operator_spaces',
+        'line_after_namespace' => 'blank_line_after_namespace',
+        'blankline_after_open_tag' => 'blank_line_after_opening_tag',
+        'return' => 'blank_line_before_return',
+        'spaces_cast' => 'cast_spaces',
+        'short_tag' => 'full_opening_tag',
+        'newline_after_open_tag' => 'linebreak_after_opening_tag',
+        'alias_functions' => 'no_alias_functions',
+        'join_function' => 'no_alias_functions',
```

Comment:
Our aliases aren't about keeping php-cs-fixer compatibility. It's about keeping compatibility with older styleci config files. The aliases do not map php-cs-fixer 1.x to 2.x.